Makefile

```diff
@@ -1,34 +1,43 @@
 # For people who don't want to use go's standard settings
 
-.PHONY: lit lit-af glit test tests
+.PHONY: lit lit-af glit test clean
 
 GOBIN = $(shell pwd)
 GO ?= latest
 WEBUI_REPO = "https://github.com/josephtchung/webui"
-all: lit test
 
+all: lit lit-af
```

Comment:
all should make `test` as well? That's how `bitcoind`'s make works, so lets do something similar.